model/Core/Vocabularies/RelationshipType.md

```diff
@@ -20,46 +20,46 @@ Build Profile specific RelationshipType descriptions can be found [here](https:/
 
 ## Entries
 
-- amends: TODOdescription
-- ancestor: TODOdescription
+- amends: The `to` element(s) amends the `from` element
```

Comment:
maybe easier to read could be:
```suggestion
- amends: Every `to` element amends the `from` element
```

Reply:
Agree - I just updated all the descriptions.